data/src/main/java/com/teamagam/gimelgimel/data/message/repository/SelectedMessageRepository.java

```diff
@@ -25,10 +25,7 @@ public SelectedMessageRepository() {
     }
 
     public void select(Message message) {
-        if (!isCurrentlySelected(message)) {
-            mCurrentlySelected = message;
-            mSelectedMessageRepo.add(message);
-        }
+        mSelectedMessageRepo.add(message);
```

Comment:
This change left the mCurrentlySelected field and isCurrentlSelected method irrelevant
remove them